nextroute/schema/input.go

```diff
@@ -67,6 +68,7 @@ type Vehicle struct {
 	MaxStops                *int            `json:"max_stops,omitempty"`
 	MaxDistance             *int            `json:"max_distance,omitempty"`
 	MaxDuration             *int            `json:"max_duration,omitempty"`
+	MaxWait                 *int            `json:"max_wait,omitempty"`
```

Comment:
Both in Cloud Fleet and Router the max wait was defined on each stop, not on each vehicle. I believe we should preserve this compatibility? Or, should we define max wait for _both_ stops and vehicles and the constraint takes care of calculating which of the max waiting times are being violated?

Reply:
Yep, that is correct. This is an incremental change and should be backwards compatible, which is why I am fine with it. :joy_cat: 
It was a comment by @davidrijsman , since he expects customers to more likely require a constraint on accumulated wait (i.e., don't make one driver wait all their shift) instead of single stop wait (don't wait more than X at a single stop - which is potentially a strange constraint). Anyway, I don't have a strong opinion here, but think it's useful and doesn't harm.

Reply:
Oh, and maybe there was something confusing here: the max_wait was already defined on the stop. It was just not implemented under the hood yet. So, you won't see it as a diff in my PR.

Reply:
Ok, more documentation for me 😅 .

Reply:
Haha, actually I wanted to ping you already: do you have a docs project somewhere? Feel free to add an issue for me to add docs for this feature here. I just don't know yet where the docs will go for this (maybe you have them already though - a bit out of the loop on docs here :grimacing: ).